core/src/main/java/org/n52/iceland/ogc/gml/time/TimePosition.java

```diff
@@ -75,9 +69,8 @@ public TimePosition(TimeIndeterminateValue indeterminateValue) {
      *            Time format
      */
     public TimePosition(DateTime time, TimeFormat timeFormat) {
-        super();
-        this.time = time;
-        this.setTimeFormat(timeFormat);
+        this.time = Optional.of(time);
+        this.timeFormat = Optional.ofNullable(timeFormat);
```

Comment:
OK. One last comment: there is a difference between `new TimePosition(time)` and new `TimePosition(time, null)`... Is `timeFormat` really a `Optional`, if you have a default value? Has `timeFormat = null` a different meaning?

Wouldn't ```Optional.ofNullable(timeFormat).orElse(DEFAULT_TIME_FORMAT)``` be more consistent?